# Discussion on Ocean Temperatures & Algae

Unfortunately, more data needs to be collected to see if the heatwave has significantly increased, or decreased the amount of time where various algae species benefit or suffers from the effect of the heatwave. 

In [3]:
library(tidyverse)
options(repr.plot.width=10, repr.plot.height=6,warn=-1)

In [23]:
temperature_data <- read_csv("data/temp_processed_summarized.csv")

Parsed with column specification:
cols(
  datetime = col_datetime(format = ""),
  mean_ext = col_double()
)


## Potential Increase in Invasive Species?

Invasive species, like *Sargassum horneri*, are known to be able to have vast, negative effects to the ecosystems they settle in. As mentioned in the study by [Sadie L. Small and Matthew S. Edwards in 2021](), *Sargassum horneri* is an invasive species that was first reported in Long Beach Harbor, CA, USA in 2003. The researchers discuss the broader impact of the displacement of native seaweeds, since many species of macroalgae are **"ecosystem-engineers"** providing a foundation for the community centered around them. If their range and presense is altered by the a more **competitive dominant invasive species**, it could have consequences like reducing **species richness** among other factors that make up the ecosystem.

This is especially concerning for coastal ecosystems around Vancouver, because it is known to live in shallow rocky reef habitats around Santa Cruz Island (Small & Edwards, 2021). Another [study by Lindsay M. Marks, Daniel C. Reed and Sally J. Holbrook in 2018](https://doi.org/10.3391/ai.2018.13.3.03) concludes that they occupied all hard substrate type, with a preference for medium and small boulders, which BC coast is no stranger to. 

With their tolerance higher than 20°C, and optimal temperatures landing around the temperatures of the heatwave, this species could be one of concern for BC's coastal waters.

In [20]:
between_15_20 <- filter(temperature_data, 
                        between(ext_temp, 15, 20))
count(between_15_20)$n / count(temperature_data)$n

[1] 0.4445686

Above calculates the percentage of datapoints that were found to be between 15 ~ 20 (not including), and it finds that **44.5%** of the data points to fit the range of this invasive species. 

## Potential Increase in Toxic Species?

Dinoflagellates in the genus *Alexandrium*  can produces paralytic shellfish toxins (toxic blooms) to a level that is unsafe for human consumption, which can lead to loss of life and to economic losses in the shellfish harvesting industry (Brian D. Bill et. al., 2016). 

As found in the study, they thrive in conditions between 13-17°C, but suffer in temperatures above 24°C. Running the same percentage calculations as above results in:

In [21]:
between_13_17 <- filter(temperature_data, 
                        between(ext_temp, 13, 17))
count(between_13_17)$n / count(temperature_data)$n

[1] 0.6042639

and finds that 60.4% of the data points fit the parameters. 

## Potential Decrease in Native Species?

Around the same temperature where *Alexandrium* starts to decline in vitality, so does *Phaeodactylum tricornutum* (Kräbs & Büchel, 2011). Although not an immediate concern, spikes in temperatures from events like the heatwave could harm the population of *Phaeodactylum tricornutum* Vancouver's strain. 

One of particular concern could be of the *Calliarthron tuberculosum* species, where Guenther & Martone found that aerial exposure and warm submersion stressed it out. With as extreme heatwave of the one like during 2021 summer on the west coast, it could dry out low tide tide pools and warm them up during high tide, which could lead to the decline of *Calliarthron tuberculosum*.